examples/analog_out/cont_gen_voltage_wfm_int_clk_non_regen.py

```diff
@@ -0,0 +1,72 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to continuously generate an
+analog output waveform by providing new data to the output buffer
+as the task is running.
+
+This example is useful if you want to generate a non-repeating waveform,
+make updates on-the-fly, or generate a frequency that is not an
+even divide-down of your sample clock. In this example,
+the default frequency value is 17.0 to demonstrate that non-regenerative output
+can be used to create a signal with a frequency that is not an even divide-down
+of your sample clock.
+"""
+
+import numpy as np
+import numpy.typing
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, RegenerationMode
+
+
+def create_sine_wave(
+    frequency: float, amplitude: float, sampling_rate: float, phase_in: float, phase_out: float
+) -> numpy.typing.NDArray[numpy.double]:
+    """Generate a sine wave."""
+    number_of_samples = int(sampling_rate * (phase_out - phase_in) * np.pi)
```

Comment:
This highlights the fact that the way these example handle number_of_samples is backwards, compared to the AO examples for LV, C, CVI, etc.

In the AO examples for the other ADEs, number_of_samples is a parameter provided by the user. Here, you are not letting the user specify a number of samples. Instead, you are calculating the number of samples based on one period of the signal you are generating.

This approach is functional for low signal frequencies, but it has significant performance problems for high signal frequencies.

For example, if you are generating a 10000 Hz signal using a 1 MS/s sample clock, then each period is 10 samples and you must call DAQmxWrite 10000 times per second. This is unsustainable.

It is better to accept number_of_samples as a parameter and use np.linspace instead of np.arange. This applies to all of the examples that generate sine waves, but it has the biggest impact on the non-regen example.

Perhaps something like this? (untested)
```
duration_time = number_of_samples / sampling_rate
duration_radians = duration_time * 2 * np.pi
phase_out = (phase_in + duration_radians) % (2 * np.pi)
x = np.linspace(phase_in, phase_in + duration_radians, number_of_samples)
return amplitude * np.sin(frequency * x)
```

Reply:
Oops, duration_radians is wrong.

Dimensional analysis:
- duration_time is in seconds (S / (S/s) = s)
- frequency is in Hz = s/cycle
- duration in cycles is duration_time (s) / frequency (s/cycle) = cycles
- duration in radians is duration in cycles * 2pi radians/cycle

so I think `duration_radians = (duration_time / frequency) * 2 * np.pi`

Reply:
Thank you for the explanation and the suggestion. I run the example with the latest changes on physical module (PXIe-6363), the scope shows sine wave signals are generated for 1 second duration and then flat line for 10 seconds (sometimes longer) until another sine wave signals are generated.
I am unfamiliar with non-regeneration example, I'm not sure is the outcome from this example is correct or not. Somehow, I don't get same sine wave signal from LabVIEW example (Voltage (non-regeneration) - Continuous Output.vi).

Generated signal from python example:
![image](https://github.com/ni/nidaqmx-python/assets/27721199/90914ca9-89f8-4c9d-b5f0-fa9b0a229bb0)

Generated signal from LabVIEW example:
![image](https://github.com/ni/nidaqmx-python/assets/27721199/b96d1303-5705-44ab-b488-3d86a2327f8c)